prospector/finder.py

```diff
@@ -134,7 +134,7 @@ def get_minimal_syspath(self, absolute_paths=True):
             if dirname not in packages:
                 module_list.append(dirname)
 
-        full_list = sorted(set(module_list) | package_list, key=len)
+        full_list = sorted(set(module_list) | package_list | {self.rootpath}, key=len)
```

Comment:
I feel like the `module_list` should already include python files from `cwd` but I'm not sure about that. I'll have a play around tomorrow to see why that isn't already happening.

Reply:
any news on that @carlio ?